### PR TITLE
SwiftPM archiving hotfix

### PR DESCRIPTION
**Pull request by kylewludwig**
Hotfix in response to an issue with archiving projects using the MAVSDK-Swift SwiftPM dependency remotely. This fix has only been tested with a local SwiftPM import; I will need to test a remote import from this fork.
- Rename `MAVSDK_Swift` import to `Mavsdk` and include _MavsdkServer.swift_.
- Create separate `MavsdkServer` import for the `mavsdk_server` binary target import to fix archiving issues. **NOTE** Currently, a Libary in Package.swift can support multiple `.target` or one `.binaryTarget`.
- Bash scripts in tools still need to be tested and fixed due to path changes. This may be a good time to bump from v0.35 to v0.36, too.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -21,7 +25,7 @@ let package = Package(
     .package(url: "https://github.com/ReactiveX/RxSwift.git", from: "5.0.0"),
   ],
   targets: [
-    .target(name: "MAVSDK-Swift",
+    .target(name: "Mavsdk",
```
Comment (JonasVautherin) [inline Package.swift:28]: Shouldn't this target now depend on `mavsdk_server`? Could it be the reason why the build fails in CI?

**Review by JonasVautherin — CHANGES_REQUESTED** [review]

File: Package.swift (hunk 1 above)
Comment (kylewludwig) [reply Package.swift:28]: I just added it to try it but there may be an issue with binary targets as dependencies in other targets. We may have to create a separate Swift Package just for the binary dependency in another GitHub repo.

**Comment by kylewludwig** [comment]
@JonasVautherin The scripts to generate the protos and docs are failing, can you take a look?

File: Package.swift (hunk 1 above)
Comment (kylewludwig) [reply Package.swift:29]: Once a release is tagged on [MAVSDK-XCFramework](https://github.com/mavlink/MAVSDK-XCFramework) for v0.37.0 this should be ready to merge. It should be tested with the [MAVSDK-Swift-Example](https://github.com/mavlink/MAVSDK-Swift-Example) by adding [my fork](https://github.com/kylewludwig/MAVSDK-Swift) using branch [swiftpm-archiving-hotfix](https://github.com/kylewludwig/MAVSDK-Swift/tree/swiftpm-archiving-hotfix)

**Review by JonasVautherin — APPROVED** [review]
Tested to work (could archive) with the MAVSDK-Swift-Example!

Thanks a lot! :rocket:

**Pull request merged**